### PR TITLE
Update Mii struct

### PR DESCRIPTION
**Pull request by hax0kartik**
Updated according to this page:- https://www.3dbrew.org/wiki/Mii

### CODE REVIEW

File: libctru/include/3ds/services/frd.h [hunk 1]
```diff
@@ -27,25 +27,34 @@ typedef struct
...
+	u32 eye_details;
+	u32 eyebrow_details;
+	u16 nose_details;
+	u16 mouse_details;
+	u16 mustache_details;
+	u16 beard_details;
+	u16 glasses_details;
+	u16 mole_details;
+	u16 author_name[10];
+}MiiData;
```
Comment (fincs) [inline libctru/include/3ds/services/frd.h]: Put back space: `} MiiData`

Comment (hax0kartik) [reply libctru/include/3ds/services/frd.h]: Fixed

**Comment by WinterMute** [comment]
Why has this been closed?

**Comment by fincs** [comment]
Superseded by #428

**Pull request closed without merging**